forward ctrl-accel keypresses to application, even when a dlg is open (e.g. useful for custom animation pane)

From: Thorsten Behrens <thb@openoffice.org>


---

 vcl/source/window/dlgctrl.cxx |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)


diff --git vcl/source/window/dlgctrl.cxx vcl/source/window/dlgctrl.cxx
index 1e9a1e2..d03489c 100644
--- vcl/source/window/dlgctrl.cxx
+++ vcl/source/window/dlgctrl.cxx
@@ -790,7 +790,9 @@ BOOL Window::ImplDlgCtrl( const KeyEvent& rKEvt, BOOL bKeyInput )
                 pWindow = pWindow->GetWindow( WINDOW_NEXT );
             }
         }
-        else
+        else if ( !aKeyCode.IsMod1() ) // no ctrl pressed - those are
+                                       // application-wide shortcuts
+                                       // we want to let pass
         {
             xub_Unicode c = rKEvt.GetCharCode();
             if ( c )
